api/analyzers/javascript/analyzer.py

```
@@ -0,0 +1,220 @@
+import io
+import os
+from ..utils import *
```

Ruff (0.8.2)

`from ..utils import *` used; unable to detect undefined names

(F403)